app/src/lib/git/status.ts

```diff
@@ -103,78 +124,35 @@ export async function getStatus(
   }
 
   const stdout = result.output.toString('utf8')
+  const parsed = parsePorcelainStatus(stdout)
+  const headers: ReadonlyArray<IStatusHeader> = parsed.filter(isStatusHeader)
```

Comment:
This can be updated to `parsed.filter(x => x.kind === 'header')` since the `isStatusHeader` function is only checking for this condition.

Reply:
see https://github.com/desktop/desktop/pull/5683#discussion_r222136803